### PR TITLE
Template params

### PR DESCRIPTION
**Pull request by ADmad**
Refs #6759

I'll update other widgets, once current changes are reviewed.

### CODE REVIEW

File: src/View/StringTemplate.php [hunk 1]
```diff
@@ -231,6 +231,11 @@ public function format($name, array $data)
         if ($template === null) {
             return '';
         }
+
+        if (isset($data['templateParams'])) {
```
Comment (markstory) [inline src/View/StringTemplate.php]: templateVars? :bike:

**Comment by markstory** [comment]
Nice work so far @ADmad. This was going to be one of my weekend tasks, but you beat me to it. Should this go into 3.1 or do you think its small enough to go into a bugfix release?

File: src/View/StringTemplate.php (hunk 1 above)
Comment (ADmad) [inline src/View/StringTemplate.php]: I used the name suggested in the original issue, but I too found myself typing `templateVars` at times so will change :)

**Comment by ADmad** [comment]
I think it's simple enough but will move it to 3.1 since it's round the corner anyway.

**Pull request closed without merging**